app/code/Magento/Customer/Model/Plugin/GetListCustomerGroupExcludedWebsite.php

```diff
@@ -64,7 +64,10 @@ public function afterGetList(
                     $customerGroupId = (int)$customerGroup->getId();
                     if (array_key_exists($customerGroupId, $allExcludedWebsites)) {
                         $excludedWebsites = $allExcludedWebsites[$customerGroupId];
-                        $customerGroupExtensionAttributes = $this->groupExtensionInterfaceFactory->create();
+                        $customerGroupExtensionAttributes = $customerGroup->getExtensionAttributes();
+                        if ($customerGroupExtensionAttributes === null) {
```

Comment:
`getExtensionAttributes` comment block contain information that function could return `null`, but in fact it never be `null`, see https://github.com/mage-os/mageos-magento2/blob/2.4-develop/lib/internal/Magento/Framework/Api/AbstractExtensibleObject.php#L169-L171
I believe that factory creation could be removed.

Reply:
Additional idea - since the interface declares it can be null, there might be a case when the custom extension can override the value to null for some reason, and this code will start failing, so maybe having this fallback to object creation might be not that bad thing.

Reply:
@ihor-sviziev I'm totally agree that interface method declaration should be fixed as weel. Good point for Magento Core PR, but it's backward incompatible changes that will produce long acceptance process by the Core Team.

> Additional idea - since the interface declares it can be null, there might be a case when the custom extension can override the value to null for some reason, and this code will start failing, so maybe having this fallback to object creation might be not that bad thing.

I have seen many modules but don't faced with situation where it replace extension attributes to null. So I think we can stay with `$customerGroupExtensionAttributes = $customerGroup->getExtensionAttributes();`